core/server/errors/database-not-seeded-error.js

```diff
@@ -0,0 +1,11 @@
+function DatabaseNotSeeded(message) {
+    this.message = message;
+    this.stack = new Error().stack;
+    this.statusCode = 500;
+    this.errorType = this.name;
+}
+
+DatabaseNotSeeded.prototype = Object.create(Error.prototype);
+DatabaseNotSeeded.prototype.name = 'DatabaseNotSeeded';
```

Comment:
My plan is to merge #7477 first, which would require this to be changed. Tell me if you prefer to do this a different way around.